src/Raven.Client/Json/BlittableJsonWriter.cs

```diff
@@ -553,7 +553,16 @@ public override void WriteValue(object value)
             switch (value)
             {
                 case BlittableJsonReaderObject readerObject:
-                    _manualBlittableJsonDocumentBuilder.WriteEmbeddedBlittableDocument(readerObject);
+                    if (false == readerObject.HasParent)
+                    {
+                        _manualBlittableJsonDocumentBuilder.WriteEmbeddedBlittableDocument(readerObject);
+                    }
+                    else
+                    {
+                        throw new NotImplementedException("Can't write value of non root blittable");
```

Comment:
`NotSupportedException` is probably better, not impl usually means that we'll get around to that.

Reply:
In fact, I meant to go back to it. But should it be not supported?